### PR TITLE
Allow bulk writes

### PR DESCRIPTION
**Pull request by djzort**
This is for #181

There are other bulk operations, but I havent attempted to chase them down. All i needed is bulk writes

### CODE REVIEW

File: sgconfig/sg_action_groups.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ DATA_ACCESS:
   - "indices:data/*"
 WRITE:
   - "indices:data/write*"
+  - "indices:data/write/bulk*"
```
Comment (floragunncom) [inline sgconfig/sg_action_groups.yml]: "indices:data/write*" includes "indices:data/write/bulk*", so no real benefit from adding this?

**Comment by djzort** [comment]
without it, we werent able to do bulk inserts. try for yourself, see if you get a different result

**Comment by floragunncom** [comment]
i cannot reproduce this, for me "indices:data/write*" works also for _bulk request. I added a testcase for this, can you review it? https://github.com/floragunncom/search-guard/commit/668e5d2cdfd1d9404918fbdf495b9e6a0c078dfc

Would be interesting to see your logfiles on DEBUG level when https://github.com/floragunncom/search-guard/issues/181 occurs

**Comment by djzort** [comment]
Ok youre right. After exhaustive testing I can no longer replicate the problem. perhaps my sg2 configuration wasnt loaded correctly...

**Comment by hex2a** [comment]
I have the same/a similar issue, this error appears if 'indices:data/write/bulk' is not explicitly added:
`[INFO ][com.floragunn.searchguard.configuration.PrivilegesEvaluator] No perm match for User [name=logstash, roles=[]] [IndexType [index=logstash-2017.02.22, type=APACHE]] [Action [indices:data/write/bulk]] [RolesChecked [sg_logstash]]`

Version: 2.4.4.11

sg_action_groups:
```
WRITE:
  - "indices:data/write*"
[..]
LOGSTASH_WRITE:
  - CREATE_INDEX
  - CRUD
```
sg_roles:
```
sg_logstash:
  indices:
    logstash-*:
      "*":
      - LOGSTASH_WRITE
```

**Pull request closed without merging**